apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/AbstractR4ResourceProvider.java

```diff
@@ -151,4 +154,63 @@ public T read(@IdParam IdType claimId, RequestDetails requestDetails) {
    */
   @VisibleForTesting
   abstract Optional<ResourceTypeV2<T>> parseClaimType(String typeText);
+
+  @Search
+  @Trace
+  public Bundle findByPatient(
+      @RequiredParam(name = "mbi")
+          @Description(shortDefinition = "The patient identifier to search for")
+          ReferenceParam mbi,
+      @RequiredParam(name = "type")
+          @Description(shortDefinition = "A list of claim types to include")
+          String type,
+      @OptionalParam(name = "isHashed")
```

Comment:
How will the caller know if this should be set to true or false?

Reply:
oh nm, `isHashed` will be a query param

Reply:
Yep, and it "defaults" to true, being that if it's not explicitly "false" (either by not being set at all or being set to something else) then it's considered true in the method logic, so under normal use cases, it's completely omitted by the caller.